src/dapp/libexec/dapp/dapp---find-libraries

```diff
@@ -13,11 +13,9 @@ set -e
 # into Hevm.  It can more accurately determine which libraries are
 # needed (via looking at the bytecode and the ASTs).
 
-function binfile() { sed -E "s/.*:(.*)/$DAPP_OUT\\/\\1.bin/"; }
 mapfile -t sources < <(find "${DAPP_SRC?}" "$DAPP_LIB" -name '*.sol')
 grep -EH '^library\s+\w+' "${sources[@]}" \
   | sed -E 's/:library[[:blank:]]+([_[:alnum:]]+)[[:blank:]]+(\{|is).*/:\1/' \
   | sort | while read -r line; do
-    # Only include libraries that have actually been built.
-    if [[ -f $(binfile <<<"$line") ]]; then echo "$line"; fi
+    echo "$line";
```

Comment:
have you tested this with some projects that use libraries?

Reply:
Only a toy example, which I will add to `dapp-test`. Still needs `--make-library-state` to work tho